src/core/Dropdown/Dropdown.baseStyles.tsx

```diff
@@ -7,15 +7,17 @@ export const baseStyles = withSuomifiTheme(
     & > [data-reach-menu-button].fi-dropdown_button {
       ${inputButton({ theme })}
       position: relative;
-      padding-right: 30px;
+      padding: 7px 38px 7px 7px;
       text-align: left;
+      line-height: 1.5;
       background-color: ${theme.colors.whiteBase};
+      box-shadow: ${theme.shadows.dropdownShadow};
       cursor: pointer;
       &:before {
         content: '';
         position: absolute;
         top: 50%;
-        right: 10px;
+        right: 16px;
```

Comment:
Would it make sense to use `insetXl` token here to get the 16px?
Or is the idea to change this after the updated tokens are released?